src/utils.ts

```diff
@@ -0,0 +1,49 @@
+type HotjarCommand = 'identify' | 'stateChange';
+
+interface WindowWithHotjar extends Window {
+  hj?: (method: HotjarCommand, ...data: unknown[]) => void;
```

Comment:
What is the reason to have this interface?

Reply:
we need to tell TS that window can have a `hj` attribute, which is a method that can have a command and other parameters